spinoso-string/src/lib.rs

```diff
@@ -54,190 +53,33 @@ pub use raw_parts::RawParts;
 mod center;
 mod chars;
 mod codepoints;
+mod enc;
 mod encoding;
 mod eq;
 mod impls;
 mod inspect;
 mod iter;
+mod ord;
```

Comment:
`OrdError` and `InvalidCodepointsError` were both pulled into their respective mods (mostly just to keep `lib` clean of any Trait implementations. Everything here is direct methods now.